superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -273,6 +328,14 @@ class Dashboard extends React.PureComponent {
       <React.Fragment>
         <OmniContianer impressionId={impressionId} dashboardId={id} />
         <DashboardBuilder />
+        <ChartModal showModal={this.showModal}
+          chart={this.modalChart}
+          addFilter={this.modalAddFilterHandler.bind(this)}
```

Comment:
if we convert modalAddFilterHandler in es6 arrow function, in that case no need to bind with this..

Reply:
done, In class we need to bind this functions with variable in constructor and then we can use variable directly in template.